Makefile

```diff
@@ -30,6 +30,7 @@ build:
 build-cross: LDFLAGS += -extldflags "-static"
 build-cross:
 	CGO_ENABLED=0 gox -output="_dist/{{.OS}}-{{.Arch}}/{{.Dir}}" -osarch='$(TARGETS)' $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' k8s.io/helm/cmd/helm
+	CGO_ENABLED=0 gox -output="_dist/{{.OS}}-{{.Arch}}/{{.Dir}}" -osarch='$(TARGETS)' $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' k8s.io/helm/cmd/tiller
```

Comment:
I would rather not do this for the following reasons:

- it vastly inflates the size of the package -- sometimes over double the original package size
- the only use case for running Tiller locally (rather than in-cluster) is (as far as I know) development, and in that case you tend to build from source.
- I think it's confusing for users to find a binary `tiller` file installed locally, when it's not intended to be run locally